browser/src/map/Map.js

```diff
@@ -952,6 +952,36 @@ L.Map = L.Evented.extend({
 		return this._zoom;
 	},
 
+	getZoomPercent: function() {
+		let zoomPercent = 100;
+		switch (this._zoom) {
```

Comment:
um - surely we already have this code in online [!] - to work out what that means. The purpose of moving it from C++ was to not duplicate this code. Please see browser/src/control/Control.StatusBar.js - and re-factor the code out of there =)

also see browser/src/map/Map.js ' getScaleZoom and the log formula in there - this code should be scared - while being very careful about precision.

Thanks!